README.md

```diff
@@ -12,6 +12,14 @@
 
 Modernizr tests which native CSS3 and HTML5 features are available in the current UA and makes the results available to you in two ways: as properties on a global `Modernizr` object, and as classes on the `<html>` element. This information allows you to progressively enhance your pages with a granular level of control over the experience.
 
+## Breaking changes with v4
+
+- These tests got removed:
+
+    - `touchevents`: [discussion](https://github.com/Modernizr/Modernizr/pull/2432) 
+    - `unicode`: [discussion](https://github.com/Modernizr/Modernizr/issues/2468) 
+    - `templatestrings` and `es6/string-templates`: merged into `es6/templatestrings` 
+
```

Comment:
a readme is an odd place for this - blog post and changelog seems more correct, no?

Reply:
- the changelog.md is generated automatically and will have that info too
- there will be a blogpost of course, but that's another repo as you know
this is just for the nerds that like readmes :-)

Reply:
ok